packages/flutter_fuchsia/lib/src/dart/fuchsia_dart_vm.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:io';
+
+import 'package:json_rpc_2/json_rpc_2.dart' as json_rpc;
+import 'package:meta/meta.dart';
+import 'package:logging/logging.dart';
+import 'package:web_socket_channel/io.dart';
+
+const Duration _kConnectTimeout = const Duration(seconds: 30);
+
+const Duration _kReconnectAttemptInterval = const Duration(seconds: 3);
+
+const Duration _kRpcTimeout = const Duration(seconds: 5);
+
+final Logger _log = new Logger('FuchsiaDartVm');
+
+/// Simple JSON RPC peer with a Fuchsia Dart VM service instance. Wraps most
+/// methods around a JSON RPC peer class.
+class FuchsiaDartVm {
+  final json_rpc.Peer _peer;
+
+  FuchsiaDartVm._(this._peer);
+
+  /// Attempts to connect to the given [Uri]. Throws an error if unable to
+  /// connect.
+  static Future<FuchsiaDartVm> connect(Uri uri) async {
+    final Stopwatch timer = new Stopwatch()..start();
+    if (uri.scheme == 'http') uri = uri.replace(scheme: 'ws', path: '/ws');
```

Comment:
The contents of the `if` block should go on the next line (or better, use the `{ }` block). Also check out our addendum to the Dart style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo.

Reply:
I think I'll just put it in brackets.  All of this code autoformats itself with dart-fmt when I write.

Reply:
Done.